README.md

```diff
@@ -4,7 +4,7 @@ This is Python package provides for quick, easy parsing for Cacophony
 Project Thermal Video (CPTV) files. It works with **Python 3** only.
 
 For more details on the internals of CPTV files, see the
-[specification](https://github.com/TheCacophonyProject/go-cptv/blob/master/SPEC.md).
+[specification](https://github.com/TheCacophonyProject/go-cptv/blob/master/SPECv2.md).
```

Comment:
This also supports v1 at the moment and we need it too (else machine learning will be hard....so we should put that here???